sdk/containerregistry/Azure.Containers.ContainerRegistry/src/Blob/BlobHelper.cs

```diff
@@ -29,6 +30,13 @@ internal static string ComputeDigest(Stream stream)
             }
         }
 
+        internal static string ComputeDigest(BinaryData data)
```

Comment:
Returning string here is a perf overhead. Since method is used only to compare with server-side digest, I'd rather pass it here as a parameter and instead of calling `FormatDigest`, compare the bytes

Reply:
Tracking with https://github.com/Azure/azure-sdk-for-net/issues/33614